modules/machine-node-custom-partition.adoc

```diff
@@ -0,0 +1,294 @@
+
+The procedure in this section shows how to add a new {op-system-first} compute node with an instance that uses a different device name from what was configured at installation. You create a custom user data secret and configure a new machine set. These steps are specific to an AWS cluster. The principles apply to other cloud deployments also. However, the device naming schema is different for other deployments and should be determined on a per-case basis.
+
+.Procedure 
+
+. On a command line, change to the `openshift-machine-api` namespace:
++
+[source,terminal]
+----
+$ oc project openshift-machine-api
+----
+
+. Create a new secret from the `worker-user-data` secret: 
+
+.. Export the `userData` section of the secret to a text file: 
++
+[source,terminal]
+----
+$ oc get secret worker-user-data --template='{{index .data.userData | base64decode}}' | jq > userData.txt
+----
+
+.. Edit the text file to add the `storage`, `filesystems`, and `systemd` stanzas for the partitions you want to use for the new node. You can specify any link:https://coreos.github.io/ignition/configuration-v3_2/[Ignition configuration parameters] as needed.
++
+[NOTE]
+====
+Do not change the values in the `ignition` stanza. 
+====
++
+[source,terminal]
+----
+{
+  "ignition": {
+    "config": {
+      "merge": [
+        {
+          "source": "https:...."
+        }
+      ]
+    },
+    "security": {
+      "tls": {
+        "certificateAuthorities": [
+          {
+            "source": "data:text/plain;charset=utf-8;base64,LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURFRENDQWZpZ0F3SUJBZ0lJTjRGN3RORnlWMGd3RFFZSktvWklodmNOQVFFTEJRQXdKakVTTUJBR0ExVUUKQ3hNSmIzQmxibk5vYVdaME1SQXdEZ1lEVlFRREV3ZHliMjkwTFdOaE1CNFhEVEl5TURFeE56RTVOREF4T1ZvWApEVE15TURFeE5URTVOREF4T1Zvd0pqRVNNQkFHQTFVRUN4TUpiM0JsYm5Ob2FXWjBNUkF3RGdZRFZRUURFd2R5CmIyOTBMV05oTUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUF3OFBRODR2SkM4WVkKL3lQK3lSeHpoMWdKa2RHcmJVRVNNzUnh5c0l6YW0rczFISGVtcS90MWN3SEh1NCtrRTNVVllCanlpdE9=="
+          }
+        ]
+      }
+    },
+    "version": "3.2.0"
+  }
+}
+
+        }
+      ]
+    },
+    "security": {
+      "tls": {
+        "certificateAuthorities": [
+          {
+            "source": "data:text/plain;charset=utf-8;base64,.....=="
+          }
+        ]
+      }
+    },
+    "version": "3.2.0"
+  },
+  "storage": {
+    "disks": [
+      {
+        "device": "/dev/nvme1n1", <1>
+        "partitions": [
+          {
+            "label": "var",
+            "sizeMiB": 50000 <2>
```

Comment:
@mburke5678 While using this I found a little typo, the trailing comma `,` is missing.

Reply:
Should be _**"sizeMiB": 50000,**_?

Reply:
Correct! :)

Reply:
Fixed: https://github.com/openshift/openshift-docs/pull/43028

Reply:
Cool, thanks